### PR TITLE
Fixed Fluent plural forms always choosing the default entry

### LINKED ISSUE #20882: Fluent plural forms are broken
The quickest way to reproduce is

https://github.com/OpenRA/OpenRA/blob/6af14c16c921468e29318915aa027e8bdf0cd807/mods/common/languages/en.ftl#L535-L538

will never hit the `[one]` case.

### PR DESCRIPTION
**Pull request by Mailaender**
Closes https://github.com/OpenRA/OpenRA/issues/20882.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -25,6 +25,7 @@ jobs:
       - name: Check Code
         run: |
           make check
+          make tests
```
Comment (PunkPun) [inline .github/workflows/ci.yml:28]: should we add this to the windows workflow as well?

**Review by PunkPun — COMMENTED** [review]
LGTM. Fixes https://github.com/OpenRA/OpenRA/pull/20876#discussion_r1199724766

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Mailaender) [reply .github/workflows/ci.yml:28]: It is already added to the Windows workflow. Do you mean the Windows PowerShell script?

Comment (PunkPun) [reply .github/workflows/ci.yml:28]: yes

File: OpenRA.Test/OpenRA.Game/FluentTest.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+using System;
+using NUnit.Framework;
+
+namespace OpenRA.Test
+{
+	[TestFixture]
+	public class FluentTest
+	{
+		readonly string pluralForms = @"
```
Comment (abcdefg30) [inline OpenRA.Test/OpenRA.Game/FluentTest.cs:20]: Can we use the file instead of this?

Comment (PunkPun) [reply OpenRA.Test/OpenRA.Game/FluentTest.cs:20]: we should use both perhaps?

Comment (Mailaender) [reply OpenRA.Test/OpenRA.Game/FluentTest.cs:20]: I wrote them in line with the .yaml unit tests.

Comment (abcdefg30) [reply OpenRA.Test/OpenRA.Game/FluentTest.cs:20]: What is the file used for?

Comment (Mailaender) [reply OpenRA.Test/OpenRA.Game/FluentTest.cs:20]: It is a temporary file as I required syntax highlighting to fix an error of mine.

**Review by abcdefg30 — APPROVED** [review]

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/cbf3441201d5a8680a3973e94f6e2b56e61e84b8...569570e9116645c1bc4073550d56437b75ac2d4c)

**Pull request merged**